Gems/Atom/Tools/PassCanvas/Code/Source/Window/PassCanvasMainWindow.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <AtomToolsFramework/Document/AtomToolsDocumentInspector.h>
+#include <AtomToolsFramework/Document/AtomToolsDocumentMainWindow.h>
+#include <AtomToolsFramework/EntityPreviewViewport/EntityPreviewViewportSettingsInspector.h>
+#include <AtomToolsFramework/EntityPreviewViewport/EntityPreviewViewportToolBar.h>
```

Comment:
I noticed this in a few headers now but I believe many of these headers can be replaced by forward declarations instead